Source/Urho3D/Graphics/AnimationController.cpp

```diff
@@ -991,7 +991,8 @@ AnimatedAttributeReference AnimationController::ParseAnimatablePath(ea::string_v
         }
 
         const ea::string_view nodePath = path.substr(0, sep);
-        animatedNode = startNode->FindChild(nodePath);
+        //Empty node path references start node in case when path starts with /@
+        animatedNode = nodePath.empty() ? startNode : startNode->FindChild(nodePath);
```

Comment:
What happens if you remove this line?
If no tests fail, then you need to either add a test or remove this line.
I'm not 100% sure we need it